packages/core/src/utils/ans/utils/ans-name-to-ibc-token-name.ts

```diff
@@ -0,0 +1,6 @@
+import { CHAIN_SEPARATOR } from '../constants'
+export function getIbcTokenName(chainName: string, asset: string) {
```

Comment:
When are we using `getX` vs just `x`? Seems inconsistent between here and `lpTokenName` based on `stakingContract`.

Reply:
`getX` is now used strictly for actions.

Action name now starts with `get` or with the actual action, like `create`, `remove` etc.